### PR TITLE
Added Go subcommand: `job queue`.

### PR DESCRIPTION
**Pull request by rgo3**
https://jira.mesosphere.com/browse/DCOS_OSS-4211

### CODE REVIEW

File: pkg/metronome/types.go [hunk 1]
```diff
@@ -119,6 +119,16 @@ type Schedule struct {
 	NextRunAt               string `json:"nextRunAt"`
 }
 
+// Queue contains all queued runs of a Job.
+type Queue struct {
+	JobID string `json:"jobId"`
+	Runs  []queuedRun `json:"runs"`
+}
+
+type queuedRun struct {
```
Comment (ArmandGrillet) [inline pkg/metronome/types.go]: What's the purpose of having a struct just to contain a string? Wouldn't `runIds  []string` in `Queue` be enough?

**Comment by br-lewis** [comment]
@mesosphere-ci run all integration tests

**Comment by rgo3** [comment]
@mesosphere-ci run all integration tests

**Comment by ArmandGrillet** [comment]
@mesosphere-ci run all integration tests

**Comment by ArmandGrillet** [comment]
@rgo3 Can you please rebase?

**Pull request merged**